docs/src/components/DocNavigation/Category.tsx

```diff
@@ -15,7 +11,7 @@ export default function Category({ name, children }: Props) {
   return (
     <StyledCategoryContainer>
       <StyledCategoryName>{name}</StyledCategoryName>
-      <StyledCategoryItems>{children}</StyledCategoryItems>
+      {children}
     </StyledCategoryContainer>
   );
 }
```

Comment:
These orbit.kiwi changes seem related to #3098, not this PR. I guess it doesn't matter if they are going to cancel themselves out after merging #3098, but I'm curious why they're here.